common/pref_names.h

```diff
@@ -39,6 +39,7 @@ extern const char kReferralHeaders[];
 extern const char kReferralAndroidFirstRunTimestamp[];
 extern const char kHTTPSEVerywhereControlType[];
 extern const char kNoScriptControlType[];
+extern const char kReduceLanguageEnabled[];
```

Comment:
seems like this should go in brave shields pref_names

Reply:
Fixed in https://github.com/brave/brave-core/pull/12234/commits/e937302cb6925c96fb2ea026788eb4409d2e325c